packages/fetchai/skills/tac_control_contract/skill.yaml

```diff
@@ -32,7 +32,7 @@ models:
     args:
       min_nb_agents: 2
       money_endowment: 2000000
-      nb_goods: 10
+      nb_goods: 11
```

Comment:
need to think about this

Reply:
see above, this should be reverted to 10, as the use of a single game currency is implicit.

Reply:
The reason this is 11 and not 10 is that when we are using the generate method in the contract we are checking if we created the items. If we create them separate and not once then we have problem with the assertion.

` assert self.token_ids == {}, "Item ids already created."`

Reply:
I'd suggest using a different way to check that then. There aren't 11 goods so this should be reverted.